README.md

```diff
@@ -22,13 +22,6 @@ permissionless validator and challenge system.
 For more detailed information about Kroma, check [Kroma docs](https://docs.kroma.network) (or
 [spec docs](./specs)).
```

Comment:
Rather, this should be fixed. Please replace to `https://specs.kroma.network` for spec docs.